tests/RequestBuilderTest.php

```diff
@@ -40,6 +40,6 @@ public function testCreateMultipartStream()
         $this->assertContains('username', $body);
         $this->assertContains('Stockholm', $body);
         $this->assertContains('city', $body);
-        $this->assertRegExp('|^multipart/form-data; boundary=[0-9a-z]+$|si', $contentType);
+        $this->assertRegExp('|^multipart/form-data; boundary=[\w\.\'()+,\-:=?]{1,69}$|si', $contentType);
```

Comment:
If you want to be strictly correct you should use the following regex: 
`^multipart/form-data; boundary=(?P<quote>"?)[\w\Q'()+,-./:=?\E]{1,69}(?P=quote)$`

Print screen form regex101.com:
<img width="716" alt="screen shot 2017-02-18 at 09 18 36" src="https://cloud.githubusercontent.com/assets/1275206/23091655/498de6bc-f5bb-11e6-830b-d84b51edc4bb.png">